sdk/iot/Azure.Iot.Hub.Service/src/IoTHubServiceClient.cs

```diff
@@ -81,5 +155,24 @@ public IoTHubServiceClient(Uri endpoint, IoTHubServiceClientOptions options)
             Files = new FilesClient();
             Jobs = new JobsClient();
         }
+
+        /// <summary>
+        /// Gets the scope for authentication/authorization policy.
+        /// </summary>
+        /// <param name="endpoint">The IoT Hub service instance Uri.</param>
+        /// <returns>List of scopes for the specified endpoint.</returns>
+        internal static string[] GetAuthorizationScopes(Uri endpoint)
+        {
+            Argument.AssertNotNull(endpoint, nameof(endpoint));
+            Argument.AssertNotNullOrEmpty(endpoint.AbsoluteUri, nameof(endpoint.AbsoluteUri));
+
+            // TODO: GetAuthorizationScopes for IoT Hub
+            return null;
+        }
+
+        private static Uri buildEndpointUriFromHostname(string hostname)
```

Comment:
A spot check of C# libraries shows "host name" as two words, so casing would be `hostName`.

Reply:
ah, got it. will change everywhere.